### PR TITLE
Feature/add workspace commands

### PR DESCRIPTION
**Pull request by KPilnacek**

### CODE REVIEW

File: elisctl/lib/api_client.py [hunk 1]
```diff
@@ -122,6 +124,28 @@ def get_paginated(self, path: str, query: Dict[str, str]) -> Tuple[List[dict], i
...
 
+    def _sideload(
+        self, objects: List[dict], sideloads: Optional[Iterable[str]] = None
+    ) -> List[dict]:
+        for sideload in sideloads or []:
+            sideloaded, _ = self.get_paginated(sideload)
+            sideloaded_dicts = {
+                sideloaded_dict["url"]: sideloaded_dict for sideloaded_dict in sideloaded
+            }
+            key = sideload.rstrip("es")
```
Comment (qiq) [inline elisctl/lib/api_client.py:135]: So key for "queues" is "queu"?

Comment (KPilnacek) [reply elisctl/lib/api_client.py:135]: Yep, but not a problem for now, as discussed IRL.

Comment (pasky) [reply elisctl/lib/api_client.py:135]: I actually don't understand why it's not a problem either. Seems like it is worth at least a code comment.

**Review by pasky — APPROVED** [review]
I don't want to hold the merge because of a single detail right now but let's come back to the `.rstrip("es")` thing.

**Pull request merged**